VMEncryption/main/SupportedOS.json

```diff
@@ -1,20 +1,9 @@
 {
   "redhat": [
         {
-          "Version" : "8.5"
-        },
-        {
-          "Version" : "8.4"
-        },
-        {
-          "Version" : "8.3"
-        },
-        {
-          "Version" : "8.2"
-        },
-        {
-          "Version" : "8.1"
-        },    
+          "Version" : "8",
+          "MinSupportedVersion" : "8.1"
+        },   
```

Comment:
can we do this with centOS as well?

Reply:
Online encryption changes are not yet fully enabled for CentOS. Also, I think CentOS is no longer in active development hence I avoided doing the change for CentOS. I will revisit it in future if there is demand from customers.

Reply:
Online encryption changes are not yet fully enabled for CentOS. Also, I think CentOS is no longer in active development hence I avoided doing the change for CentOS. I will revisit it in future if there is demand from customers.